cpp/oneapi/dal/algo/knn/common.cpp

```diff
@@ -19,9 +19,32 @@
 #include "oneapi/dal/exceptions.hpp"
 
 namespace oneapi::dal::knn {
+
+namespace result_options {
+
+template <typename Task>
+const inline result_option_id_t default_result_options = responses;
+
+template <>
+const inline result_option_id_t default_result_options<task::search> = indices | distances;
+
+} // namespace result_options
```

Comment:
Need to remove this namespace as does not add value

Reply:
fixed